RadClass/RadClass.py

```diff
@@ -206,7 +233,7 @@ def iterate(self):
         # number of samples analyzed between log updates
         log_interval = max(min((self.stop_i - self.start_i)/100, 100), 10)
         running = True  # tracks whether to end analysis
-        while running:
+        while running and (self.start_i != self.stop_i):
```

Comment:
This allows `RadClass` to still run without throwing an error, but essentially process no data if `self.start_i == self.stop_i`.
The new condition in the `while` statement is required, otherwise the `pbar.update` below would throw an error (because `inverse_dt = nan`).